File: problem-test/src/main/java/com/dudgns/problemtest/repository/carManagement/CarCategoryMappingRepository.java
package com.dudgns.problemtest.repository.carManagement;

import com.dudgns.problemtest.entity.carManagement.CarCategoryMappingEntity;
import com.dudgns.problemtest.entity.id.CarCategoryMappingId;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface CarCategoryMappingRepository extends JpaRepository<CarCategoryMappingEntity, Long> {
    List<CarCategoryMappingEntity> findByCarCategoryMappingId(CarCategoryMappingId carCategoryMappingId);
}

File: problem-test/project-car-mgt-api/adapter-data-jpa/src/main/kotlin/dudgns/com/backend/carmgt/adapter/data/jpa/repository/carManagement/CarCategoryMappingRepository.kt
package dudgns.com.backend.carmgt.adapter.data.jpa.repository.carManagement

import dudgns.com.backend.commons.data.entity.problemTest.CarCategoryMappingEntity
import org.springframework.data.jpa.repository.JpaRepository

interface CarCategoryMappingRepository : JpaRepository<CarCategoryMappingEntity, Long> {
    fun findByCarCategoryMappingIdCarIdx(carIdx: Long?) : List<CarCategoryMappingEntity>
}